### PR TITLE
feat: graphql mutations skeleton for chat functionality

### PR DESCRIPTION
**Pull request by JenniferFalkenstein**

### CODE REVIEW

File: graphql/me/fields.ts [hunk 1]
```diff
@@ -3,7 +3,19 @@ import { getSessionUser, GraphQLUser } from '../authentication';
 import { GraphQLContext } from '../context';
 import { Role } from '../authorizations';
 import { UserType } from '../types/user';
+import { Field, ObjectType } from 'type-graphql';
 
+@ObjectType()
+class Contact {
```
Comment (Jonasdoubleyou) [inline graphql/me/fields.ts:9]: Can't we use the User (UserType) here?

Comment (LomyW) [reply graphql/me/fields.ts:9]: I think that the `Contact` type would maybe extended by fields e.g. `contactReason`, if we add a own `contact` table.

Comment (Jonasdoubleyou) [reply graphql/me/fields.ts:9]: Well how about:

class Contact {
  user: UserType;
  contactReason: ...;
  chatId: string;
}

File: graphql/me/fields.ts [hunk 2]
```diff
@@ -17,4 +27,16 @@ export class FieldMeResolver {
     myRoles(@Ctx() context: GraphQLContext): string[] {
         return context.user?.roles ?? [];
     }
+
+    @Query((returns) => [String])
```
Comment (Jonasdoubleyou) [inline graphql/me/fields.ts:31]: [Contact] ?

**Review by Jonasdoubleyou — APPROVED** [review]

**Pull request merged**